doc/docs/index.md

```diff
@@ -4,12 +4,27 @@ nRF Connect Programmer is an app available from [nRF Connect for Desktop](https:
 
 ## Supported devices
 
-- Nordic Thingy:91
-- Nordic Thingy:52
-- nRF91 Series DKs
-- nRF53 Series DKs
-- nRF52 Series DKs and Dongle
+The following table lists the available programming methods for Nordic Semiconductor devices. The devices with no programming method supported are deprecated.
 
-### Deprecated support
-
-- nRF51 Series DKs and Dongle
+|                     | PCA number | Programming with J-Link  | Programming with Nordic Secure DFU  | Programming with MCUboot |
+|---------------------|------------|--------------------------|-------------------------------------|--------------------------|
+| nRF9161 SiP         | PCA10153   | Yes                      | No                                  | No                       |
+| nRF9160 SiP         | PCA10090   | Yes                      | No                                  | No                       |
+| nRF9131 SiP         | PCA10147   | Yes                      | No                                  | No                       |
+| Nordic Thingy:91    | PCA20035   | Yes                      | No                                  | Yes                      |
+| nRF7002 DK          | PCA10143   | Yes                      | No                                  | No                       |
+| nRF5340 DK          | PCA10095   | Yes                      | No                                  | No                       |
+| nRF5340 Audio DK    | PCA10121   | Yes                      | No                                  | No                       |
+| Nordic Thingy:53    | PCA20053   | Yes                      | No                                  | Yes                      |
+| nRF52840 DK         | PCA10056   | Yes                      | No                                  | No                       |
+| nRF52840 Dongle     | PCA10059   | No                       | Yes                                 | No                       |
+| nRF52833 DK         | PCA10100   | Yes                      | No                                  | No                       |
+| nRF52832 DK (nRF52) | PCA10040   | Yes                      | No                                  | No                       |
+| nRF52820 DK         | PCA10115   | No                       | No                                  | No                       |
+| nRF52811 DK         | PCA10068   | No                       | No                                  | No                       |
+| Nordic Thingy:52    | PCA20020   | No                       | No                                  | No                       |
+| nPM1300 EK          | PCA10152   | Yes                      | No                                  | Yes                      |
```

Comment:
The EK Does nor have a built In Jlink only an external jlink can be used so not sure if this should be true or false

Reply:
Edited.